### PR TITLE
typos, wording for import.Rmd

### PR DESCRIPTION
**Pull request by datalove**
Hi Hadley, I made a few changes here, fixing some typos that I found. I've also proposed a few minor changes where I thought the wording could be improved for clarity.

### CODE REVIEW

File: import.Rmd [hunk 1]
```diff
@@ -65,7 +67,7 @@ Most of readr's functions are concerned with turning flat files into data frames
   [webreadr](https://github.com/Ironholds/webreadr) which is built on top 
   of `read_log()`, but provides many more helpful tools.)
 
-readr also provides a number of functions for reading files off disk into simpler data structures:
+readr also provides a number of functions for reading files from disk into data structures simpler than a data frame:
```
Comment (datalove) [inline import.Rmd]: Not sure whether the convention is to capitalise `readr` if it's the start of a sentence. In either case, best to be consistent.

File: import.Rmd [hunk 2]
```diff
@@ -136,7 +138,7 @@ Parsing occurs after leading and trailing whitespace has been removed (if not ov
 
 #### Datetimes
 
-Readr provides three options depending on where you want a date (the number of days since 1970-01-01), a date time (the number of seconds since midnight 1970-01-01), or a time (i.e. the number of seconds since midnight). The defaults read:
+readr provides three options depending on whether you want a date (the number of days since 1970-01-01), a date time (the number of seconds since midnight 1970-01-01), or a time (i.e. the number of seconds since midnight). The defaults are:
```
Comment (datalove) [inline import.Rmd]: Again... how to capitalise `readr` when it starts a sentence?

File: import.Rmd [hunk 3]
```diff
@@ -12,7 +12,9 @@ library(readr)
 
 ## Overview
 
-You can't apply any of the tools you've applied so far to your own work, unless you can get your own data into R. In this chapter, you'll learn how to import:
+To apply the tools we've discussed in your work, you'll first have to get your own data into R.
+
```
Comment (datalove) [inline import.Rmd]: The sentence feels like it was a bit of a mouthful. I've trimmed it down a bit, and hopefully haven't lost much of the meaning.

File: import.Rmd [hunk 4]
```diff
@@ -23,18 +25,18 @@ The common link between all these packages is they all aim to take your data and
 
 ## Flat files
 
-There are many ways to read flat files into R. If you've be using R for a while, you might be familiar with `read.csv()`, `read.fwf()` and friends. We're not going to use these base functions. Instead we're going to use `read_csv()`, `read_fwf()`, and friends from the readr package. Because:
+There are many ways to read flat files into R. If you've be using R for a while, you might be familiar with `read.csv()`, `read.fwf()` and friends. We're not going to use these functions from base R. Instead we're going to use `read_csv()`, `read_fwf()`, and friends from the readr package because:
```
Comment (datalove) [inline import.Rmd]: 'base R' sounds a bit better to my ear than just 'base', but imagine that's quite subjective :)

File: import.Rmd [hunk 5]
```diff
@@ -23,18 +25,18 @@ The common link between all these packages is they all aim to take your data and
...
 
-* These functions are typically much faster (~10x) than the base equivalents.
-  Long run running jobs also have a progress bar, so you can see what's
+* These functions are typically much faster (~10x) than the base R equivalents.
+  Long running jobs also have a progress bar, so you can see what's
```
Comment (datalove) [inline import.Rmd]: Again with 'base R' versus 'base'

File: import.Rmd [hunk 6]
```diff
@@ -23,19 +25,18 @@ The common link between all these packages is they all aim to take your data and
...
   it's slightly less flexible than readr, but can be twice as fast.)
   
 * They have more flexible parsers: they can read in dates, times, currencies,
   percentages, and more. 
   
-* They fail to do some annoying things like converting character vectors to 
-  factors, and munging the column headers to make sure they're valid R 
-  variable names.
+* They refrain from doing annoying things like converting character vectors to 
+  factors. And don't mangle the column names when headers aren't valid R variable names.
```
Comment (datalove) [inline import.Rmd]: Is 'mangle' more appropriate here than 'mung'?

**Comment by hadley** [comment]
I still need to rewrite this chapter a few times before it's ready for comments. But I appreciate you taking a look!

**Pull request closed without merging**